raft/README.md

```diff
@@ -26,7 +26,8 @@ This raft implementation is a full feature implementation of Raft protocol. Feat
 - Log compaction 
 - Membership changes
 - Leadership transfer extension
-- Lease-based linearizable read-only queries served by both the leader and followers
+- Effecient linearizable read-only queries served by both the leader and followers
+- More effecient lease-based linearizable read-only queries served by both the leader and followers
```

Comment:
Be clear about what "efficient" and "more efficient" mean here (or somewhere). Mentioning followers sounds like this is a purely local operation, but followers still need to talk to the leader (so it spreads the load a little bit but for lowest latency it's still important to talk to the leader).